docs/fastddsgen/dataTypes/dataTypes.rst

```diff
@@ -463,73 +463,122 @@ User annotations will be passed to TypeObject generated code if the ``-typeobjec
-+-------------------------+--------------------------------------------------------------------------------------------+
-| @verbatim               | [Unimplemented] Allow injecting some user-provided information into what the compiler |br| |
-|                         | will generate.                                                                             |
-+-------------------------+--------------------------------------------------------------------------------------------+
-| @service                | [Unimplemented] Allow indicating that an interface is to be treated as a service.          |
-+-------------------------+--------------------------------------------------------------------------------------------+
-| @oneway                 | [Unimplemented] Allow indicating that an operation is one way only, meaning that |br|      |
-|                         | related information flow will go from client to server but not back.                       |
-+-------------------------+--------------------------------------------------------------------------------------------+
-| @ami                    | [Unimplemented] Allow indicating that an interface or an operation is to be made |br|      |
-|                         | callable asynchronously.                                                                   |
-+-------------------------+--------------------------------------------------------------------------------------------+
-| @non_serialized         | The annotated member will be omitted from serialization.                                   |
-+-------------------------+--------------------------------------------------------------------------------------------+
-
-Most unimplemented annotations are related to Extended Types.
+.. _builtin_annotations:
+
+Additionally, the following standard annotations are defined by the specification and considered builtin (these
+annotations might be applied without the need of defining them).
+
+.. list-table::
+    :header-rows: 1
+    :align: left
+
+    * - Builtin Annotation
+      - Behavior
+      - Supported
+    * - :code:`@ami`
+      - Asynchronous interface or operation.
+      - ❌
+    * - :code:`@appendable`
+      - Shortcut for :code:`@extensibility(APPENDABLE)`.
+      - ✅
+    * - :code:`@autoid`
+      - Member ID algorithm configuration if there is no member ID explicitly set using :code:`@id` annotation. |br|
+        Possible values are :code:`SEQUENTIAL` (member ID is assigned sequentially) or |br|
+        :code:`HASH` (member ID is calculated with an algorithm involving hashing the member name). |br|
+        This annotation might be defined in module, structure or union declarations.
+      - ✅
+    * - :code:`@bit_bound`
+      - Set number of bits on `bitmasks`_ and `enumerations`_ underlying primitive type. |br|
+        Currently, :code:`@bit_bound` can only be applied to bitmask types.
+      - ✅❌
```

Comment:
```suggestion
      - ❎
```

Reply:
I love it but I think we should leave it as it is for now.

Reply:
😢